scheduler/src/cook/mesos/api.clj

```diff
@@ -39,6 +39,8 @@
             [cook.mesos.unscheduled :as unscheduled]
             [cook.mesos.util :as util]
             [cook.mesos]
+            [cook.rate-limit :refer [job-submission-rate-limiter]]
+            [cook.rate-limit.generic :as rt]
```

Comment:
Why `rt`?

Reply:
TODO (per our discussion earlier): Use :rtg instead of :rt for rate-limit-generic in unit tests. Outside of unit tests, we'll have cook.rate-limit re-export everything in cook.rate-limit.generic, so there will be no explicit import of cook.rate-limit.generic Also, no :refer for job-submisison-rate-limiter.

Reply:
If we use @DaoWen's code snippet to re-`def` everything from `rate-limit.generic` into `rate-limit`, then we shouldn't even need to `require` `rate-limit.generic` here.

Reply:
The code snippet didn't work, but I imported the 4 needed functions manually.